.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+  "trailingComma": "es5",
+  "semi": false,
```

Comment:
Per the poll in Slack so far, I think _including_ semicolons wins.

I think we should remove the trailingComma and semi options here so that we use the latest defaults. But I do like single quotes, so let's leave that option...!